### PR TITLE
[FEAT] Add Dzongkha

### PR DESCRIPTION
**Pull request by KinWang-2013**
This PR adds Dzongkha, the national Language of Bhutan 🇧🇹

### CODE REVIEW

File: gem/locales/dz.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# :one_other pluralization (see https://github.com/ddnexus/pagy/blob/master/gem/lib/pagy/i18n.rb)
+en:
```
Comment (benkoshy) [inline gem/locales/dz.yml:2]: should this be a `dz`?

**Comment by benkoshy** [comment]
Hi @KinWang-2013 

Thank you for your PR

- Please confirm that this is a `:one_other` pluralization? I couldn't find `dz` [in the list here](https://github.com/svenfuchs/rails-i18n/tree/master/rails/pluralization) and was wanting to double check to be sure?

File: gem/locales/dz.yml (hunk 1 above)
Comment (KinWang-2013) [reply gem/locales/dz.yml:2]: oh yeah, thanks

**Comment by KinWang-2013** [comment]
> Hi @KinWang-2013
> 
> Thank you for your PR
> 
> * Please confirm that this is a `:one_other` pluralization? I couldn't find `dz` [in the list here](https://github.com/svenfuchs/rails-i18n/tree/master/rails/pluralization?rgh-link-date=2024-10-04T01%3A00%3A25Z) and was wanting to double check to be sure?

yes it is. I will make a PR there as well

File: gem/lib/pagy/i18n.rb [hunk 2]
```diff
@@ -113,6 +113,7 @@ module P11n
                  hash['zh-CN'] = RULE[:other]
                  hash['zh-HK'] = RULE[:other]
                  hash['zh-TW'] = RULE[:other]
+                 hash['dz']    = RULE[:other]
```
Comment (ddnexus) [inline gem/lib/pagy/i18n.rb:116]: This line is inconsistent with the pluralization indicated in the `gem/locales/dz.yml` and its entries.

File: gem/locales/dz.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# :one_other pluralization (see https://github.com/ddnexus/pagy/blob/master/gem/lib/pagy/i18n.rb)
```
Comment (ddnexus) [inline gem/locales/dz.yml:1]: Inconsistent with the `:other` pluralization indicated in the `gem/lib/pagy/i18n.rb` entry

File: gem/lib/pagy/i18n.rb (hunk 2 above)
Comment (KinWang-2013) [reply gem/lib/pagy/i18n.rb:116]: thank you, will update

File: gem/locales/dz.yml (hunk 3 above)
Comment (KinWang-2013) [reply gem/locales/dz.yml:1]: thank you, will update

**Comment by ddnexus** [comment]
> yes it is. I will make a PR there as well

From the entries in the dictionary, it doesn't look that the pluralization rule is `:one_other`, since they show the same translation. It looks more like `:other`. Please, link an official page for the `dz` locale that shows which pluralization rule applies.

Thank you.

**Comment by KinWang-2013** [comment]
> > yes it is. I will make a PR there as well
> 
> From the entries in the dictionary, it doesn't look like the pluralization rule is `:one_other`, since they show the same translation. It looks more like `:other`. Please, link an official page for the `dz` locale that shows which pluralization rule applies, and fix either the rule or the entries.
> 
> Thank you.

sorry, got a bit confused, will update, I don't think there are any official page for dz locale.

**Comment by ddnexus** [comment]
Maybe we can link https://www.unicode.org/cldr/charts/45/supplemental/language_plural_rules.html at the top of the `dz` dictionary file.

**Comment by ddnexus** [comment]
Thank you

**Pull request merged**